### PR TITLE
PagerDuty integration review and revisions

### PR DESCRIPTION
**Pull request by tpdorsey**
## Summary
Revisions based on review and testing of PagerDuty integration process.

Affects: https://docs.prefect.io/orchestration/integrations/pagerduty.html

## Changes
* Some rewriting for clarity and style.
* Fix broken internal links to other topics.
* Clarify requirements, including supported Prefect Cloud plans.
* Clarify RBAC usage.
* Installation and configuration process tested and confirmed. Minor edits for clarity.
* Added image showing action configuration in Prefect Cloud.
* Confirmed and clarified support contacts.
* Additional minor changes for consistency.

## Importance
Corrects errors in documented requirement and steps, and fixes broken links.

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: docs/orchestration/integrations/pagerduty.md [hunk 1]
```diff
@@ -1,55 +1,74 @@
 # PagerDuty + Prefect
 
-## Integration Benefits
+Using the PagerDuty integration with Prefect, you can receive PagerDuty alerts based on events raised by your Flow runs and Agents.
```
[the remark this answers is not in the document]
Comment (madkinsz) [reply docs/orchestration/integrations/pagerduty.md:3]: This is definitely an objective we had settled on for internal objects.

[the remark this answers is not in the document]
Comment (tpdorsey) [reply docs/orchestration/integrations/pagerduty.md]: Got it. Fixed in commit

File: docs/orchestration/integrations/pagerduty.md [hunk 2]
```diff
@@ -1,55 +1,74 @@
...
+First, make sure you have a configured PagerDuty service to connect with Prefect. From the **Configuration** menu in PagerDuty, select **Services**.
+
+There are two ways to add an integration to a PagerDuty service:
+
+* **If you are adding your integration to an existing service**: Click the **name** of the service you want to add the integration to. Then, select the **Integrations** tab and click the **New Integration** button.
+* **If you are creating a new service for your integration**: Please read the PagerDuty documentation [Configuring Services and Integrations](https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations) and follow the steps outlined in the [Create a New Service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting **PREFECT** as the **Integration Type** in step 4. Continue with the [In Prefect](#in-prefect) section below once you have finished these steps.
+
+Enter an **Integration Name** in the format `monitoring-tool-service-name` (for example, "PREFECT-FLOW-AUTOMATION-EXAMPLE") and select **PREFECT** from the **Integration Type** menu.
```
[the remark this answers is not in the document]
Comment (tpdorsey) [reply docs/orchestration/integrations/pagerduty.md]: Fixed in new commit

**Review by anna-geller — APPROVED** [review]
nice!

**Comment by tpdorsey** [comment]
Pushed minor change related to Kingsley feedback.

**Comment by kingsleyb** [comment]
These changes look good to me.

**Review by madkinsz — APPROVED** [review]

**Pull request merged**